templates/footer.html

```diff
@@ -0,0 +1,5 @@
+				</div>
```

Comment:
Take a look at the last commit. I'm leaving only `<body>` open, so header.html includes only `<head>` and `<nav>`